packages/eslint-plugin/src/rules/no-unsafe-enum-comparison.ts

```diff
@@ -66,6 +66,19 @@ export default util.createRule({
       'BinaryExpression[operator=/=|<|>/]'(
         node: TSESTree.BinaryExpression,
       ): void {
+        // Exempt bit shift operators since they are not comparisons
+        //
+        // ```ts
+        // enum Fruit {
+        //  Apple,
+        // }
+        //
+        // const myBitShift = 1 << Fruit.Apple;
+        // ```
```

Comment:
[Nit] I think this comment is pretty clear from the code & tests. WDYT?

```suggestion
```

Reply:
Please see [3d812c6](https://github.com/typescript-eslint/typescript-eslint/pull/7074/commits/3d812c69909794f8ede28c717cf86fbae5cabf63) where I've removed the if statement along with the comment